package.json

```diff
@@ -17,11 +17,12 @@
         "pretest": "npm run lint",
         "test": "jest",
         "migrate": "tsx ./build/run-migration.ts",
-        "lint": "npm run lint:ts && npm run lint:json && npm run prettier:scss",
+        "lint": "npm run svelte-check && npm run lint:ts && npm run lint:json && npm run prettier:scss",
```

Comment:
Why do we need to run `svelte-check` if we're also adding a new svelte-eslint package?

Reply:
I thought `eslint-typescript` didn't work because `no-undef` errors for globals like `game` and `fu` were generated in svelte files. I just noticed that `no-undef` is also off in the recommended `typescript-eslint` options so I'll recheck if it actually works.

Reply:
I've tested with the following svelte file with the current config:
```svelte
<script lang="ts">
    type Test = Array<string>;

    //Test type checking
    game.i19n.localize("test");
</script>

```
<img width="698" alt="eslint-svelte-errors" src="https://github.com/user-attachments/assets/5b748596-7e11-47bd-8718-aa7017a6e8c2">

It's not picking up on the wrong `i19n` type. I actually think `typescript-eslint` doesn't pick up on type errors at all because the `game.i19n.localize("test");` isn't picked up in `npm run lint` in master either.

Running `svelte-check` will emit errors in both `.ts` and `.svelte` files:

<img width="695" alt="svelte-check" src="https://github.com/user-attachments/assets/ab1feb83-8d7e-48d0-87f5-3da19f71988f">

Reply:
The linter doesn't do type checking; building does

Reply:
yeah, but `vite-plugin-checker` (which runs `tsc` after a build unless im totally mistaken) doesn't support svelte hence the inclusion of `svelte-checker`.